packages/components/transitions/src/slide.tsx

```diff
@@ -82,53 +84,49 @@ export const slideProps = {
  *
  * @see Docs https://yamada-ui.com/components/transitions/slide
  */
-export const Slide = forwardRef<SlideProps, "div", false>(
-  (
-    {
-      unmountOnExit,
-      isOpen,
-      placement: _placement = "right",
-      transition,
-      transitionEnd,
-      delay,
-      duration = { enter: 0.4, exit: 0.3 },
-      className,
-      __css,
-      ...rest
-    },
-    ref,
-  ) => {
-    const animate = isOpen || unmountOnExit ? "enter" : "exit"
+export const Slide = forwardRef<SlideProps, "div", false>((props, ref) => {
+  const [style, mergedProps] = useComponentStyle("Slide", props)
+  const {
+    unmountOnExit,
+    placement: _placement = "right",
+    transition,
+    transitionEnd,
+    delay,
+    duration = { enter: 0.4, exit: 0.3 },
+    className,
+    __css,
+    ...rest
+  } = omitThemeProps(mergedProps)
 
-    const placement = useValue(_placement)
+  const animate = rest.isOpen || unmountOnExit ? "enter" : "exit"
 
-    const custom = { placement, transition, transitionEnd, delay, duration }
+  const placement = useValue(_placement)
 
-    isOpen = unmountOnExit ? isOpen && unmountOnExit : true
+  const custom = { placement, transition, transitionEnd, delay, duration }
 
-    const { position } = getSlideProps(placement)
+  const isOpen = unmountOnExit ? rest.isOpen && unmountOnExit : true
```

Comment:
Why was it necessary to modify this section?If you don't need to fix it, please revert back to the original.

Also, this modification will cause a warning on the console because of the `isOpen` in `rest`.